src/ticTacToe/game/Cell.java

```diff
@@ -0,0 +1,21 @@
+package ticTacToe.game;
+
+import ticTacToe.ai.MiniMax;
+
+/**
+ * Class that contain coordinate and rate of cell of field
+ *
+ * @see MiniMax
+ * @see ticTacToe.ai.SelfLearning
+ */
+public class Cell {
+    public int s;
+    public int r;
+    public int rate;
+
+    public Cell(int string, int row) {
+        this.s = string;
```

Comment:
It's not necessary to add `this.` in contexts where it can be omitted. Also, consider better names, because it's hard to understand `r` and `s` in the code there these variables used.